modules/ROOT/pages/dw-operators.adoc

```diff
@@ -653,24 +667,19 @@ payload map ((user) ->
 
 === Upserting
 
-The `update` operator enables you to insert a field if it is not present by using the `!` symbol at the end of the selector expression. When the field is not present the value `null` is going to be bound to the variable. +
-
-The following example updates the category based on the name of the user:
+The `update` operator enables you to insert a field that is not present already by using the `!` symbol at the end of the selector expression. When the field is not present, the operator binds a `null` value to the variable. 
 
-.Input
-[source,json,linenums]
-----
-[{"lastName": "Doe"}, {"lastName": "Parker", "name": "Peter" }]
-----
+The following example appends the specified key-value pair to the first object in the `myInput` array:
```

Comment:
```suggestion
The following example iterates through the objects in the Array and for each one updates the field `name` based on if it present or not. If it is not present, it will add the field `name` with the value "JHON" and if it is present, it will upper the name value.
```

Reply:
done, copyedited